_rules/heading-non-empty-accessible-name-ffd0e9.md

```diff
@@ -173,14 +175,6 @@ This `h1` element has an [explicit role][] of `none`. However, the [global][] [p
 
 #### Inapplicable Example 1
 
-There is no [semantic][semantic role] `heading` element.
-
-```html
-<div></div>
-```
```

Comment:
Removing this example breaks our own "test case design" guidelines of [Test every condition](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/test-cases.md#test-every-condition) 🤔

Reply:
@WilcoFiers No strong feeling about that particular example which is, indeed, quite useless 🙂

It's more on the boundaries of things on how we decide that tests are useful or not. 
From a TDD point of view, we should indeed have test cases for every condition, and since we do use the test cases to validate tools, this also sounds like a good idea from an engineering point of view.
From an "illustrate the rule to human" point of view, this is indeed a fairly useless case…

=> this tends to hint that we may want to have some test cases for the computers and only show a meaningful subset to the humans.